src/Engine/Engine.cpp

```diff
@@ -567,8 +567,8 @@ Engine::~Engine() {
 }
 
 void Engine::LogEngineBuildInfo() {
-    logger->info("OpenEnroth, revision {}", gitRevision());
-    logger->info("Extra build information: {}/{}/{} {}", BUILD_PLATFORM, BUILD_TYPE, BUILD_COMPILER, PROJECT_VERSION);
+    logger->info("OpenEnroth, revision {} built on {}", gitRevision(), buildTime());
```

Comment:
Reintroducing compile time does not pose problems that was encountered last time?

Reply:
Nope, this way it just works. The previous problem we had was with ccache choking on __DATE__.